rclcpp_examples/src/parameters/set_and_get_parameters.cpp

```diff
@@ -27,7 +27,7 @@ int main(int argc, char ** argv)
 
   auto parameters_client = std::make_shared<rclcpp::parameter_client::SyncParametersClient>(node);
 
-  // Set several differnet types of parameters.
+  // Set several different types of parameters.
```

Comment:
I could be wrong, though. Maybe there's the internet, the samenet and the differnet :stuck_out_tongue_winking_eye: